rest/utilities_testing.go

```diff
@@ -2306,6 +2306,18 @@ func (rt *RestTester) GetKeyspaces() []string {
 	return keyspaces
 }
 
+func (rt *RestTester) GetDbCollections() []*db.DatabaseCollection {
```

Comment:
This is worth a comment because I get asked this, it's return all collections in lexicographic order for ease of test usage and it'll match `GetKeyspaces()` or `getCollectionsForBLIP`

Reply:
`getCollectionsForBLIP` isn't sorted in the same way (its ranging over a uint-keyed map) but I've added a comment and said it matches `GetKeyspaces` at least